index.html

```diff
@@ -69,7 +69,384 @@ <h4 class="sub-title">I am a Software Developer</p>
+
+    
+    <header class="prof-art3">
+      <h1 class="art3">Professional Art</h1>
+      <h2 class="data3">Printing Data</h2>
+    </header>
+
+    <Main>
+      <div class="text4">
+      <p>
+        A daily selection of privately
+         personalized reads; no accounts
+          or sign-ups required. has been the
+           industry's standard
+      </p>
+    </div>
+    
+      <div class="program">
+
+        <ul class="prog-item3">
+          <li class="html3">html</li>
+          <li class="boot3">bootstrap</li>
+          <li class="ruby3">ruby</li>
+        </ul>
+      
+        <div class="see3">See Project</div>
+
+      </div>
+    
+    
+    </Main>
+
+  </section>
+
+  <section class="wrap4">
+
+    
+    <header class="prof-art4">
+      <h1 class="art4">Professional Art</h1>
+      <h2 class="data4">Printing Data</h2>
+    </header>
+
+    <Main>
+      <div class="text5">
+      <p>
+        A daily selection of privately
+         personalized reads; no accounts
+          or sign-ups required. has been the
+           industry's standard
+      </p>
+    </div>
+    
+      <div class="program">
+
+        <ul class="prog-item4">
+          <li class="html4">html</li>
+          <li class="boot4">bootstrap</li>
+          <li class="ruby4">ruby</li>
+        </ul>
+      
+        <div class="see4">See Project</div>
+
+      </div>
+    
+    
+    </Main>
+
+  </section>
+
+  <section class="wrap5">
+
+    
+    <header class="prof-art5">
+      <h1 class="art5">Professional Art</h1>
+      <h2 class="data5">Printing Data</h2>
+    </header>
+
+    <Main>
+      <div class="text6">
+      <p>
+        A daily selection of privately
+         personalized reads; no accounts
+          or sign-ups required. has been the
+           industry's standard
+      </p>
+    </div>
+    
+      <div class="program">
+
+        <ul class="prog-item5">
+          <li class="html5">html</li>
+          <li class="boot5">bootstrap</li>
+          <li class="ruby5">ruby</li>
+        </ul>
+      
+        <div class="see5">See Project</div>
+
+      </div>
+    
+    
+    </Main>
+
+  </section>
+
+  <section class="wrap6">
+
+    
+    <header class="prof-art6">
+      <h1 class="art6">Professional Art</h1>
+      <h2 class="data6">Printing Data</h2>
+    </header>
+
+    <Main>
+      <div class="text7">
+      <p>
+        A daily selection of privately
+         personalized reads; no accounts
+          or sign-ups required. has been the
+           industry's standard
+      </p>
+    </div>
+    
+      <div class="program">
+
+        <ul class="prog-item6">
+          <li class="html6">html</li>
+          <li class="boot6">bootstrap</li>
+          <li class="ruby6">ruby</li>
+        </ul>
+      
+        <div class="see6">See Project</div>
+
+      </div>
+    
+    
+    </Main>
+
+  </section>
+
+  <section class="head-style">
+
+    <header class="head-h1">
+      <h1>About me</h1>
+    </header>
```

Comment:
- [ ] Kindly reduce the font-size of your `h1` for it to be in line with the given template.

### Your Design

![your-about-me](https://user-images.githubusercontent.com/26597432/199761655-8907c5e7-52d9-48ca-b3df-51d3dcee1051.png)


### Figma Design

![temp-about-me](https://user-images.githubusercontent.com/26597432/199761721-c0c61460-e5a5-4960-a6f9-8c33adf65db5.png)